include/libtorrent/torrent_handle.hpp

```diff
@@ -293,8 +293,8 @@ namespace aux {
 		// omitted then it is implicitly set to the end of the file the first
 		// piece is set to. If the torrent metadata has not been downloaded yet
 		// then the functions do nothing.
-		void set_sequential_range(piece_index_t const first_piece, piece_index_t const last_piece) const;
-		void set_sequential_range(piece_index_t const first_piece) const;
+		void set_sequential_range(piece_index_t first_piece, piece_index_t last_piece) const;
+		void set_sequential_range(piece_index_t first_piece) const;
```

Comment:
Won't this break the API/ABI?

Reply:
the compiler ignores this particular use of `const` (to an `int`) in the declaration and the lint suggests removing it. It's still present in the definition.

Reply:
right. top level `const` is redundant in function declarations.